src/components/AppLoading.tsx

```diff
@@ -25,6 +28,41 @@ export const AppLoading: FC<PropsWithChildren> = ({ children }) => {
 
   const isLoading = !isLoadingComplete || isSignedIn === null || !isLayoutReady
 
+  const currentVersion = Application.nativeApplicationVersion || 'unknown'
+  const os = Platform.OS
+
+  const { mutate: checkForUpdate } = useSystemControllerCheckForAppUpdate()
+
+  useEffect(() => {
+    if (os === 'ios' || os === 'android') {
+      checkForUpdate(
+        {
+          data: {
+            currentVersion,
+            os,
+          },
+        },
+        {
+          onSuccess: (data) => {
+            if (data.updateRequired) {
+              const storeUrl =
+                os === 'ios'
+                  ? `https://apps.apple.com/app/id${data.appId}`
```

Comment:
What do you think about move this and below url to consts ?

Reply:
That's nice, thanks for the feedback, really appreciate it